### ISSUE TITLE
Why is the version of FastAPI fixed in the install_requires?

### ISSUE THREAD

**Issue by gukoff** [identixone/fastapi_contrib#141]
Currently fastapi/pydantic versions are fixed in `setup.py`:
```
requirements = [
    'fastapi==0.52.0',
    'pydantic==1.4',
    'contextvars;python_version<"3.7"'
]
```

Is there a reason not to use `>=` instead?

If compatibility is a concern, it's possible to test against different versions of FastAPI on TravisCI.

**Comment by levchik** [comment]
Hi, @gukoff 

The reason for this is that we've had some issues with fastapi and pydantic new minor versions broking code in this repo. Since both packages are pretty much mature now, I think we can remove hard version pinning and implement your suggested changes. 

I think that I will have time to make changes myself no earlier than in a week or so, but you are very welcome to send PR.

**Comment by levchik** [comment]
Solved by ccb6a40d7b008e4628a99ea36e324acd47041cb3
